### ISSUE TITLE
Can I train big version of YOLO instead of tiny-YOLO?

### ISSUE THREAD

**Issue by earcz** [deividasskiparis/aws-darknet-yolo-docker#1]
Hi,

I need to train YOLOv3 from scratch on my custom dataset with AWS. Is it possible to use your repo for that purpose ?  Thanks in advance.

**Comment by deividasskiparis** [comment]
Of course :)
1. In /config, change the .cfg and pre-trained weights file to whatever network architecture you like. Doesn't have to be 'Yolo'
2. Be sure to change ENV variables in Dockerfile according to your new filenames

If you don't have any pre-trained weights, be sure to inlcude `-e PRETRAINED_WEIGHTS=""` when running the image (Step 6.)
`docker run -d --runtime=nvidia --name=train-yolo -e PRETRAINED_WEIGHTS="" train-yolo`
Everything else should work out of the box

**Comment by earcz** [comment]
Hi Deividas,

Frist of all, thanks for your reply and so sorry for my late response to thank you.

I am also trying to figure out Step 2 which provides information about structure of folders. I want to use my custom dataset, COCO and PASCAL VOC respectively. Can you please provide a little detail about test.txt, train.txt and label folder (including image-name-1.txt) ? How can I prepare these files? Answer to my this question will help me to understand completely everything.

Thank you.

**Comment by deividasskiparis** [comment]
Hi @earcz 
The structure of folders that you see in step 2 is really just a guideline, but you can create whatever structure you want. So the key things to mention
- In `config/config.data` you will find `train` and `valid` vars. You set them with the path of text files that have your training and validation data.
- Each row in `train` and `valid` text files contains a path to an image.
- According to AlexeyAB, in [How to train (to detect your custom objects)](https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects), Step5, you then generate labels in the same dir as your images. I prefer to keep them separate from images and place them in `../labels` dir relative to images.

Hope this clarifies things :)

**Comment by earcz** [comment]
Hi @deividasskiparis 

You clarified many things concisely. Thank you :)
